doc/ref_model/chapters/chapter01.md

```diff
@@ -65,7 +65,7 @@ The document starts from the abstract and as it progresses it increasingly gets
   > **Audience**: This chapter is written for lifecycle managers, operational support teams and others who need to support the infrastructure or the workload applications.
 - **Chapter 10 - Challenges and Gaps**: Opportunities for future developments as technology changes over time.
   > **Audience**: This chapter is written for a general technical audience with interest in this topic.
-- **Appendix A - Guidelines For VNF Vendors**: More details related to how the workload applications will interface with the cloud infrastructure through APIs (including VIM and CaaS).
+- **Appendix A - Guidelines For VNF Vendors**: More details related to how the workload applications will interface with the cloud infrastructure through APIs (including Cloud Infrastructure Manager and CaaS).
```

Comment:
Is this still true of Appendix A?

Reply:
I think yes, and it should be **Appendix A - Guidelines For Application Vendors**

Reply:
Issue #1543 opened

Reply:
> 
> 
> Issue #1543 opened

This might be a duplicate, we have opened issues #1537, #1539 and #1540 to change the title and the references to the appendix.